src/compiler/commandLineParser.ts

```diff
@@ -133,6 +133,16 @@ namespace ts {
             description: Diagnostics.Raise_error_on_this_expressions_with_an_implied_any_type,
         },
         {
+            name: "noUnusedLocals",
+            type: "boolean",
+            description: Diagnostics.Variable_0_has_never_been_used,

```

Comment:
this should not be an error message, it should be a message describing the flag. something like `Raise an error if a local variable is unused`